src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/InitializeHandler.cs

```diff
@@ -31,7 +31,11 @@ internal class InitializeHandler : IRequestHandler<InitializeParams, InitializeR
                 DefinitionProvider = true,
                 DocumentHighlightProvider = true,
                 RenameProvider = true,
-                ReferencesProvider = true
+                ReferencesProvider = true,
+                SignatureHelpProvider = new SignatureHelpOptions()
+                {
+                    TriggerCharacters = new[] { "(", "," }
```

Comment:
Using the same `TriggerCharacters` as [roslyn](https://github.com/dotnet/roslyn/blob/440c6400e3d4c3eb955ae1aab098717701cfba5a/src/Features/LanguageServer/Protocol/Handler/Initialize/InitializeHandler.cs#L48).